docs/extensibility/payment-method-integration.md

```diff
@@ -166,7 +166,7 @@ A big part of the payment method integration is the interface that is exposed fo
-| `shippingData`           | Object   | Contains all shipping related data (outside of the shipping status).                                                                                                                                                                                                                                               | `shippingRates`, `shippingRatesLoading`, `selectedRates`, `setSelectedRates`, `isSelectingRate`, `shippingAddress`, `setShippingAddress`, and `needsShipping`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `shippingData`           | Object   | Contains all shipping related data (outside of the shipping status).                                                                                                                                                                                                                                               | `shippingRates`, `isLoadingRates`, `selectedRates`, `setSelectedRates`, `isSelectingRate`, `shippingAddress`, `setShippingAddress`, and `needsShipping`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Is this change still true?

Reply:
No, but a new change to this file is, there is no such method as `setSelectedRates` on `usePaymentMethodInterface` so the TS definition and docs have been updated.